docs/operate/reference/components/servo/_index.md

```diff
@@ -25,6 +25,8 @@ If you have a physical ["RC" or "hobby" servo motor](https://learn.adafruit.com/
 
 If your motor is coupled with an [encoder](/operate/reference/components/encoder/), not a potentiometer, for position feedback, you should configure it as an [encoded motor](/operate/reference/components/motor/encoded-motor/) instead.
```

Comment:
Maybe remove?

Reply:
```suggestion
```
Fair, should be somewhat self-evident plus this could confuse folks with industrial servos